README.md

```diff
@@ -21,7 +21,7 @@ This plugin is available on Pub: [https://pub.dev/packages/razorpay_flutter](htt
 Add this to `dependencies` in your app's `pubspec.yml`
 
 ```yaml
-razorpay_flutter: ^1.1.1
+razorpay_flutter: ^1.1.3
```

Comment:
Why not `1.1.2`?

Reply:
It was bumped earlier to 1.1.2 and the readme was not updated.

Reply:
My bad, I thought it was pubspec.yml